src/app/shared/main-navigation/mat-main-nav/menu-items.const.ts

```diff
@@ -83,31 +83,19 @@ export const menuItems: IMenuItem[] = JSON.parse(
     {
         "name": "Revalidation",
         "type": 0,
-        "route": "admin/manage-connections",
+        "route": "/trainees",
         "description": "",
         "menuItems": [
             {
                 "type": 0,
-                "route": "admin/manage-connections",
-                "name": "Manage connections",
+                "route": "/trainees",
+                "name": "All Trainees",
                 "description": ""
             },
             {
                 "type": 0,
-                "route": "concerns/",
-                "name": "Concerns log",
-                "description": ""
-            },
-            {
-                "type": 0,
-                "route": "admin/manage-connections/create",
-                "name": "Connection discrepancies",
-                "description": ""
-            },
-            {
-                "type": 0,
-                "route": "revalidation/",
-                "name": "Provide recommendation",
+                "route": "/trainees/under-notice",
```

Comment:
`/trainees/under-notice` doesn't exist as a route. im guessing you meant something like `/trainees?<filterParamNameHere>=under-notice`?

Reply:
Yeah thought it would after our many routing strategy discussions. Previous routes in diff didnt exist either

Reply:
alternatively this value could just be `/` which will redirect to `/trainees` without the optional params?

Reply:
ok, i'll take the navigation item out as till route exists